Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,31 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling OSTaskCreate() and provide argument specifying to RTOS how the task will be managed. 
```

Comment:
Putting `` `` around text puts the words in the cool code font - you should put them around "OSTaskCreate()"